releasenotes/notes/46136.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+
+# issue is a list of GitHub issues resolved in this note.
+issue:
+  - https://github.com/istio/istio/issues/46136
```

Comment:
nit: this can just be either `https://github.com/istio/istio/issues/46136` or `46136`